frontend/pages/index.page.tsx

```diff
@@ -90,6 +89,24 @@ export default function Home() {
           </Card>
         </Grid>
       </Grid>
+      <Grid
+        container
+        spacing={{ xs: 2, md: 3 }}
+        columns={{ xs: 12 }}
+        alignItems={"center"}
+        justifyContent={"center"}
+      >
+        <Grid item xs={12} sm={6} md={6} alignItems="center" justifyContent="center">
+          <Container>
+            <AboutModularHistory />
+          </Container>
+        </Grid>
+        <Grid item xs={12} sm={6} md={6}>
```

Comment:
@sunzae99, the featured content should be displayed before the "today in history" content, I think.